src/content/docs/iast/testing-status.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: IAST testing status page
+tags:
+    - New Relic IAST
+    - IAST
+metaDescription: Find out the data you can find on the New Relic IAST testing status page.
+redirects:
+    - /docs/iast/summary
+freshnessValidatedDate: never
+---
+
+import iastSummary from 'images/iast_screenshot-full_summary.webp'
+
+
+The New Relic IAST testing status page provides a centralized view of your tested applications, vulnerabilities, coverage, and more at a glance. Viewing vulnerabilites along with their risk scores and potential exposure helps you to prioritize remediation decisions quickly.
+
+Go to **[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST** to open the IAST testing status page.
+
+<img
+    title="IAST testing status page"
+    alt="IAST testing status page"
+    src={iastSummary}
+/>
+
+<figcaption>
+    IAST testing status page, **[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST**.
+</figcaption> 
+
+The IAST testing status page includes these sections on the left navigation pane:
+
+- **A: Overview**. When you open the [IAST UI](https://one.newrelic.com/iast/summary), the IAST testing status page opens.
```

Comment:
"When you open... the status page opens" reads a little awkwardly. It's not gating but I'd consider seeing if you can reword it a bit.